app/src/main/java/com/fivesysdev/standwithukraine/data/DayStatisticRepositoryImpl.kt

```diff
@@ -7,16 +7,15 @@ import java.util.concurrent.FutureTask
 
 class DayStatisticRepositoryImpl : DayStatisticRepository {
     private var statistics: MutableList<DayStatistic> = ArrayList()
-    var fromDate: LocalDate = LocalDate.now().with(TemporalAdjusters.firstDayOfMonth())
+    private var fromDate: LocalDate = LocalDate.now().with(TemporalAdjusters.firstDayOfMonth())
 
     init {
         updateStatistics()
     }
 
     private fun updateStatistics() {
         val task = Callable {
-            val client = Client()
-            return@Callable client.getStatisticsFromDate(fromDate.toString())
+            return@Callable StatisticService().getStatisticFromDate(fromDate.toString())
         }
         val future = FutureTask(task)
         Thread(future).start()
```

Comment:
I've just noticed the `statistics.stream()` in `addAll(list: List<DayStatistic>) ` I'm sure you can find a better way to do this:)

Reply:
Thanks, I'll rewrite this part of the code.